tests/test_wave_01.py

```diff
@@ -32,12 +29,15 @@ def test_get_tasks_one_saved_tasks(client, one_task):
     ]
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_task(client, one_task):
     # Act
     response = client.get("/tasks/1")
     response_body = response.get_json()
 
+    print("LOOK AT THIS")
+    print(response_body)
+    print("LOOK AT THIS")
```

Comment:
this made me laugh lol. but also don't forget to remove these development print tests before finalizing a project. 😌